web/client/utils/LayersUtils.js

```diff
@@ -153,17 +153,19 @@ export const getNestedGroupTitle = (id, groups = []) => {
 
 /**
  * Flatten nested array to a one-level array
- * @param {object[]} array of objects
- * @returns {object[]} flattened array
+ * @param {Object[]} array of objects
```

Comment:
@tonym3nsa where did you see that Object type needs to be in capital case?
I think we can keep it lowercase since the whole project is using this convention
probably we did this because docma lib was having issues with it
@offtherailz right?

Reply:
@MV88 Well noted. I have however noted that there are various instances in the project where `@param {object} `starts with an uppercase O. Is it something we need to take care of?

Reply:
I asked to make you try because it was originally causing problems when generating documentation. I did some tests and now it seems not to be a problem anymore (and the more correct syntax may be "Object").
So for me it's ok.